pkg/gui/gui.go

```diff
@@ -311,6 +311,21 @@ func (gui *Gui) resetState(filterPath string, reuseState bool) {
 	if filterPath != "" {
 		screenMode = SCREEN_HALF
 		initialContext = contextTree.LocalCommits
+	} else if gitArgs != "" {
+		switch gitArgs {
+		case "status":
+			screenMode = SCREEN_HALF
+			initialContext = contextTree.Files
+		case "branch":
+			screenMode = SCREEN_HALF
+			initialContext = contextTree.Branches
+		case "log":
+			screenMode = SCREEN_HALF
+			initialContext = contextTree.LocalCommits
+		case "stash":
+			screenMode = SCREEN_HALF
+			initialContext = contextTree.Stash
+		}
```

Comment:
we should terminate the program with an error if we are given an unexpected value for gitArg